portfolio/src/main/webapp/js/scripts.js

```diff
@@ -43,41 +43,47 @@
 
 
 function loadComments() {
-    fetch('/comments')
-    .then(response => response.json())
-    .then((comments) => 
+  var num = document.getElementById("number").value;
```

Comment:
nit: use let here

Reply:
Thanks, I keep forgetting about let in JavaScript :)